### ISSUE TITLE
Bumping to Zeek 6.0.0/6.0.1: Failed to run command -C

### ISSUE THREAD

**Issue by awelzel** [zeek/try-zeek#58]
I've pulled the `zeek/zeek:6.0.0` and `zeek/zeek:6.0.1` images on the tryzeek box, restarted the tryzeek service and the hello world example failed as follows. I've reverted by removing the 6.0.x images and restarting the tryzeek service again.

![Screenshot-20230920-183713](https://github.com/zeek/try-zeek/assets/2100509/b4655b5c-383c-4b56-a5b8-7ab1b158df20)

Haven't looked at the details.

**Comment by JustinAzoff** [comment]
Oh, yes!

I looked into this but forgot to make a ticket for it.

I had tried adding zeek 6, but it fails because newer glibc uses a different syscall implementation for `stat` which fails on the older kernel.  This loop is what fails:

```
for prog in /bro/bin/bro /bro/bin/zeek /usr/local/zeek/bin/zeek ; do
    if [ -x $prog ] ; then
        PROG=$prog
    fi
done
```
since the `-x` that bash runs fails.  adding a default doesn't work either, because zeek then fails for similar reasons.  We'll need to get the base system OS updated, then 6 should work.
